docs/user-guide/src/bmo/ironic_installation.md

```diff
@@ -0,0 +1,187 @@
+
+```python
+# The libvirt URI to use. This option enables libvirt drivers.
+# qemu+ssh://root@<remote-libvirt-server>/system?keyfile=<path-to-key>
+SUSHY_EMULATOR_LIBVIRT_URI = "qemu+ssh://root@192.168.111.1/system?&keyfile=/root/ssh/id_rsa_virt_power&no_verify=1&no_tty=1"
+SUSHY_EMULATOR_IGNORE_BOOT_DEVICE = False
+SUSHY_EMULATOR_VMEDIA_VERIFY_SSL = False
+```
+
+We will mount `conf.py` into the container's `/root/sushy/` path.
+
+```bash
+ sudo docker run -d --net host \
+    --name sushy-tools \
+     -v "SOME_PATH/conf.py":/root/sushy/conf.py \
+     -v "/root/.ssh":/root/ssh \
+     quay.io/metal3-io/sushy-tools:latest
+```
+
+Our sushy-tools container image is available at  [quay.io/metal3-io/sushy-tools](https://quay.io/repository/metal3-io/sushy-tools?tag=latest&tab=tags)
+
+### httpd
+
+We use this container to run an Apache web server for hosting node images. Below is the list of images (from `$IMAGES_PATH`) that we mount into the httpd container's `/shared` path.
+
+1. Ironic Python Images (IPA) via http for PXE booting
+    - ironic-python-agent.kernel
+    - ironic-python-agent.initramfs
+
+2. Target node image in qcow2 format
+    - bionic-server-cloudimg-amd64-disk.img
+
+We are not directly using Apache container image here, but instead [quay.io/metal3-io/ironic](https://quay.io/repository/metal3-io/ironic?tag=latest&tab=tags) which has `runhttpd` entry point to start off the httpd.
+
+`PROVISIONING_INTERFACE` - interface to use for ironic, dnsmasq (dhcpd) and httpd.
+
+```bash
+sudo docker run -d --net host --privileged \
+    --name httpd-infra \
+    -v "IMAGES_PATH":/shared \
+    --entrypoint /bin/runhttpd \
+    --env "PROVISIONING_INTERFACE=ironicendpoint" \
+    quay.io/metal3-io/ironic:latest
+```
+
+## Ironic in-cluster installation
+
+For in-of-cluster Ironic installation, we will run a set of containers within
+a single pod in a Kubernetes cluster. You can enable TLS or basic auth or even
+disable both for Ironic and inspector communication. Below we will see kustomize
+folders that will help us to install Ironic for each mentioned case. In each
+of these deployments, a ConfigMap will be created and mounted to the Ironic pod.
+The ConfigMap will be populated based on environment variables from
+[ironic-deployment/default/ironic_bmo_configmap.env](https://github.com/metal3-io/baremetal-operator/blob/main/ironic-deployment/default/ironic_bmo_configmap.env). As such, update
+`ironic_bmo_configmap.env` with your custom values before deploying the Ironic.
+
+We assume you are inside the local baremetal-operator path, if not you need to
+clone it first and `cd` to the root path.
+
+```bash
+ git clone https://github.com/metal3-io/baremetal-operator.git
+ cd baremetal-operator
+```
+
+TLS and Basic authentication disabled:
+
+```bash
+ kustomize build ironic-deployment/default | kubectl apply -f -
+```
+
+Basic authentication enabled:
```

Comment:
This requires additional files to be present with credentials.

Reply:
do you think just mentioning of the need for username & password is enough ?